packages/epo-widget-lib/src/widgets/LightCurvePlot/DM15Display/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { ForeignObject } from "@/charts/index";
+import { estimateMagnitude } from "../helpers";
+import * as Styled from "./styles";
+
+interface DM15DisplayProps {
+  gaussianWidth: number;
+  className?: string;
+}
+
+const DM15Display: FunctionComponent<DM15DisplayProps> = ({
+  gaussianWidth,
+  className,
+}) => {
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation();
+  const peak = estimateMagnitude(0, gaussianWidth);
+  const fifteenDay = estimateMagnitude(15, gaussianWidth);
+  const dm15 = fifteenDay - peak;
+
+  return (
+    <ForeignObject>
+      <Styled.DisplayContainer className={className}>
+        <Styled.Line />
+        <math>
+          <mn>&Delta;</mn>
+          <msub>
+            <mi>m</mi>
+            <mn>15</mn>
+          </msub>
+          <mo>=</mo>
+          <mn
+            role="status"
+            aria-live="polite"
+            aria-label={t("light_curve.deltaM15.label") || undefined}
```

Comment:
make this value live, VoiceOver struggles with reading some of the math symbols so a custom label is provided here